optimum/intel/openvino/modeling_base.py

```diff
@@ -285,7 +287,7 @@ def _from_transformers(
 
         compression_option = None
         if load_in_8bit is not None:
-            compression_option = "int8" if load_in_8bit else "fp32"
+            compression_option = "fp32"
```

Comment:
could you add a comment to specify that if `load_in_8bit` is not specified, then it'll be set as default (depending on the model size)

Reply:
There is a message about that [here](https://github.com/huggingface/optimum-intel/blob/374b1fc71301f8f60440416648114c571f7063f0/optimum/exporters/openvino/__main__.py#L308) which is the only one entry point for auto identification of 8-bit compression.

Reply:
Done

Reply:
thanks!